src/component/project/ProjectAccess/ProjectGroupView/ProjectGroupView.tsx

```diff
@@ -107,6 +114,20 @@ interface IProjectGroupViewProps {
     onRemove: () => void;
 }
 
+const useHiddenColumns = (): string[] => {
```

Comment:
Opportunity to refactor here. Would be nice to have the same pattern for all lists.

Reply:
Refactored